Makefile

```diff
@@ -21,6 +21,9 @@ deps:
 proxy:
 	go build -i -o $(PROXY) -ldflags "$(LDFLAGS)" ./proxy/proxy.go
 
+amd64:
```

Comment:
Besides building for the default OS, can we also include building binary for other OS, like Windows, Linux, darwin?

Reply:
Yes, I'm currently working on a ticket to improve the packaging for the proxy. This will be addressed in that.